packages/backend-plugin-api/src/services/definitions/coreServices.ts

```diff
@@ -277,4 +277,15 @@ export namespace coreServices {
   export const urlReader = createServiceRef<
     import('./UrlReaderService').UrlReaderService
   >({ id: 'core.urlReader' });
+
+  /**
+   * Information about the current Backstage instance.
+   *
+   * @public
+   */
+  export const instanceMetadata = createServiceRef<
+    import('./InstanceMetadataService').InstanceMetadataService
+  >({
+    id: 'core.instanceMetadata',
```

Comment:
Another one for the maintainers? Do we want to have this plugin scoped? Not sure what an instance metadata might do that's plugin specific, but maybe this has been discussed already?

Reply:
Making it plugin scoped would allow us to add plugin-specific utilities like `getOtherInstalledPlugins()` and whatnot. I don't think the value is quite there compared to the ability to be able to use this in other root services like the root HTTP router. It does raise the question of whether we should call this `rootInstanceMetadata` though, which I think we should, and update the ID.